numpy/_core/code_generators/generate_numpy_api.py

```diff
@@ -278,6 +279,10 @@ def do_generate_api(targets, sources):
         init_list.append(api_item.array_api_define())
         module_list.append(api_item.internal_define())
 
+    # In case we end with a "hole", append more NULLs
+    while len(init_list) <= unused_index_max:
+        init_list.append("        NULL")
```

Comment:
I'd write this as
```
if len(init_list) <= unused_index_max:
    init_list.extend(["        NULL"] * (unused_index_max + 1 - len(init_list)))
```
but it doesn't really matter, I guess.

Reply:
Yeah, I guess, but on the other hand, this is similar to a pattern above and really doesn't matter :).